src/functionaltests/java/com/ericsson/ei/utils/HttpPostRequests.java

```diff
@@ -0,0 +1,132 @@
+package com.ericsson.ei.utils;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class HttpPostRequests {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HttpPostRequests.class);
+    private int port;
+    private String url = "http://localhost:";
+    private String endpoint = "/subscriptions";
+    private Map<String, String> headers = new HashMap<>();
+    private StringEntity params;
+
+    private static JSONArray jsonParams = null;
+    private SubscriptionRestAPI restApi = new SubscriptionRestAPI();
+
+
+    /**
+     * Constructor for initializing a HttpPostRequest object
+     *
+     * @param applicationPort
+     *      which port to use for request
+     * @param requestUrl
+     *      url for request
+     * @param urlEndpoint
+     *      which endpoint to use
+     * */
+    public HttpPostRequests(int applicationPort, String requestUrl, String urlEndpoint) {
```

Comment:
You can simplify your constructor make it more readable 
public HttpPostRequests(int port, String url, String endpoint) {
this.port = port;
this.endpoint =endpoint;
this.url = url;
}